packages/replayio/src/utils/recordings/formatRecording.ts

```diff
@@ -1,50 +1,72 @@
-import chalk from "chalk";
 import { formatDuration, formatRelativeDate } from "../date";
+import { dim, link } from "../theme";
 import { LocalRecording } from "./types";
 
+const MAX_TITLE_LENGTH = 35;
+
 export function formatRecording(recording: LocalRecording) {
   const id = recording.id.substring(0, 8) + "…";
 
   let title;
   if (recording.metadata.host) {
-    title = chalk.blueBright.underline(recording.metadata.host);
+    if (recording.metadata.host.length > MAX_TITLE_LENGTH) {
+      title = link(recording.metadata.host.substring(0, MAX_TITLE_LENGTH) + "…");
```

Comment:
To avoid trailing whitespace followed by `...` we could do this:
```suggestion
      title = link(recording.metadata.host.substring(0, MAX_TITLE_LENGTH).trimEnd() + "…");
```